charts/landing/Chart.yaml

```
@@ -16,7 +16,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.3
+version: 0.1.4
```

Not a bug fix but a feature. Please follow semver conventions.

```suggestion
version: 0.2.0
```

---

we have two urls for one service: wardenprotocol.org and genesis.wardenprotocol.org. and we can use simple enought description for ingress. Like this
```
ingress:
  enabled: true
  className: "kong"
  annotations:
    cert-manager.io/cluster-issuer: letsencrypt
  hosts:
    - name: "wardenprotocol.org"
      paths:
        - port: 3000
    - name: "genesis.wardenprotocol.org"
      paths:
        - port: 3000
  tls:
    - hosts:
        - "wardenprotocol.org"
        - "genesis.wardenprotocol.org
```